packages/components/pie-card/test/component/pie-card.spec.ts

```diff
@@ -304,4 +284,80 @@ test.describe('PieCard - Component tests', () => {
             });
         });
     });
+
+    test.describe('Prop: `disabled" and tag="button"`', () => {
+        test('should have the correct attributes', async ({ mount, page }) => {
+            // Arrange
+            await mount(PieCard, {
+                props: {
+                    tag: 'button',
+                    disabled: true,
+                } as CardProps,
+                slots: {
+                    default: slotContent,
+                },
+            });
+
+            // Act
+            const component = page.locator(componentSelector);
+
+            // Assert
+            await expect(component).toHaveAttribute('aria-disabled', 'true');
+            await expect(component).toHaveAttribute('tabindex', '-1');
+            await expect(component).toBeDisabled();
+        });
+
+        test('should not trigger the click event when the card is a button and disabled', async ({ mount, page }) => {
+            // Arrange
+            const messages: string[] = [];
+
+            await mount(PieCard, {
+                props: {
+                    tag: 'button',
+                    disabled: true,
+                } as CardProps,
+                slots: {
+                    default: slotContent,
+                },
+                on: {
+                    click: () => messages.push('1'),
+                },
+            });
+
+            // Act
+            const component = page.locator(componentSelector);
+            // using page.evaluate to attach the click event as the playwright click method will time out if the element is disabled.
+            await page.evaluate(() => {
+                const card = document.querySelector('pie-card');
+                card?.shadowRoot?.querySelector('div')?.click();
+            });
+
+            // Assert
+            await expect(component).toBeDisabled();
+            expect(messages).toHaveLength(0);
+        });
+    });
+
+    test.describe('Prop: `disabled" and tag="a"`', () => {
```

Comment:
Place inside the main parent "disabled".

Reply:
Example: https://github.com/justeattakeaway/pie/pull/1946/files#diff-5375b8f692432c3f08dec3af2f84994cbdb7a8f84ba3123e182485876bd8472aR308